src/pip/_internal/commands/list.py

```diff
@@ -157,7 +161,8 @@ def run(self, options: Values, args: List[str]) -> int:
 
         if options.outdated and options.list_format == "freeze":
             raise CommandError(
-                "List format 'freeze' can not be used with the --outdated option."
+                "List format 'freeze' cannot be used together with the --outdated "
+                "option."
```

Comment:
Is this one personal preference? The changes seem very trivial and don’t improve much to me.

Reply:
"cannot" is based on feedback from the ticket linked in the description. Adding "together" seems to have been my personal preference at the time of creating this PR. I don't have strong feelings about this though.

Reply:
Removed the "together"